lib/screens/home.dart

```diff
@@ -112,12 +180,11 @@ class HomeState extends State<Home> with SingleTickerProviderStateMixin {
       appBar: AppBar(
         // backgroundColor: lightGreen,
         centerTitle: true,
-        title: Text(widget.appName),
+        title: const Text('KeyPod'),
```

Comment:
Why the reversion to being hardcoded?

Reply:
Fixed by using future builder (commit https://github.com/anusii/keypod/pull/67/commits/39a2294dead6caed78b8622141ff48b3baa739a9).